documentation/common/prerequisites-openshift.adoc

```diff
@@ -10,4 +10,11 @@ been tested to work with the latest stable release of the OpenShift Origin Clien
 link:https://github.com/minishift/minishift[Minishift^] for an example of how to run a local instance of OpenShift
 on your machine.
 
+* If you want to install {ProductName} on Minishift and want to use Service Catalog, you need to explicitly enable it during the start-up, like
```

Comment:
@jenmalloy Let me know how it looks now. Also, are there instructions available anywhere on how build docs locally for testing?

Reply:
@dejanb, actually, it would be ideal to run a script to build docs locally for testing, and I've set up a meeting with the Strimzi folks to see if we can reuse their script. I'll add these instructions as soon as possible. Thanks for suggesting this!